docs/api/dots.md

```diff
@@ -1,11 +1,11 @@
 ## Dots component props
 
+* ```value: Number```: Current Carousel value
+
+* ```onChange: PropTypes.func```: OnChange callback (works the same way as onChange in Carousel component)
```

Comment:
```suggestion
* ```onChange: PropTypes.func```: OnChange callback (works in the same way as onChange in Carousel component)
```

Reply:
`works in the same way` is more popular https://textranch.com/227775/it-works-the-same-way/or/it-works-in-the-same-way/